Add translation PR template (#47)
Schneegans/Fly-Pie: .github/PULL_REQUEST_TEMPLATE/add_or_update_translation.md

---
name: Add or update a translation
about: Make Fly-Pie international! 🌐
title: '<Add/Update> <LANGUAGE> translation'
labels: translation
assignees: ''
---

<!--
Thank you for contributing a translation to Fly-Pie!

> This template will help you to to provide a high quality contributon to the project.

Author: Yes, of course xD

Please replace words in pointy <brackets> with suitable replacements.

---

This block of text is a comment - it will not show up in the final version of the
Pull request, but provide additional info to you on how to do things correctly.
You should not delete them, in case you would like to look something up later.
In order to do so once you submitted the PR, click on "edit" in the top right corner.
-->

## Requirements to get this PR accepted

<!--
Make sure to comply to all requirements and tick the boxes once you have done the task.
Look at existing translations for reference. Even if your PR doesn't meet all requirements
right now, please do submit it anyways! We are a friendly community and eager to help you out.

Once all boxes are ticked, your PR is ready to be merged (providing Fly-Pie's maintainer
is ok with it - but I'm sure they will be! xD)
-->

- [ ] I followed the "Translating FlyPie" section of the README.
- [ ] I complied to the Contributing Guidelines. <!--see CONTRIBUTING.md-->

> - [ ] I made the following changes to my `<LANGUAGE>.po` file: <!--You'll find most of them in the first code block, around lines 1-20-ish.-->

Reviewer: I think it is important that the file name reflects the language code. This is actually **not the case** for the Italian translation. Maybe we should fix that! `it` or `it_IT`...?

```suggestion
- [ ] I made the following changes to my `<LANGUAGE_CODE>.po` file: <!--You'll find most of them in the first code block, around lines 1-20-ish.-->
```

Author: I guess it's just `it`, same as with `de`. Are there any other countries that speak Italian that may need a different translation from yours, @albanobattistella?

  - [ ] Fill in the `<LANGUAGE>`.
  - [ ] In case you are the first translator, fill in your info and the year. If not, just tick this box anyway.
  - [ ] Set `PO_Revision-Date` to an appropriate point in time.
        <!--It doesn't have to be accurate to the second, but at least the hour should be accurate.-->
  - [ ] Set `Last-Translator` accordingly.
        <!--Format: Firstname Lastname <my@email.address>.
        Alternatively, use your GitHub handle.-->

>   - [ ] Update `Language-Team`. If your name is already on there, just tick this box anyway.
>         <!--Please place your full name or GitHub handle at the end of the comma-seperated list.-->

Author: My thought is to use this as a kind of "mailing list" to ping people if an updated translation is necessary.
Do you have a better idea on how to handle this?

```suggestion
  - [ ] Update `Language-Team`. If your name is already on there, just tick this box anyway.
        <!--Please place your full name or GitHub handle at the end of the comma-seperated list.
        This will be used to notify you when a translation needs to be updated.-->
```

  - [ ] Set `Language` to the language code (`de_DE`, `en_GB`) or just the country code (`de`, `en`).
        <!--If there are special words that are written differently in two countries with the same
        language (e. g. "color" and "colour" in English), the first variant is obligatory.-->
  - [ ] Make sure that `charset` is set to `UTF-8`.
        <!--This allows for special characters like ä, ö, ü, è etc. to be displayed correctly.-->

## Additional ideas/questions

> 

Author: Maybe we could ask people here if they want to get notified when the translation needs to be updated?

<!--

> This is the place for any additional stuff concerning your Pull Request.
> Please put it below this comment. If there isn't anything else to say, please delete the heading.

Reviewer: Maybe like this?

```suggestion
This is the place for any additional stuff concerning your Pull Request.
For example, you could drop a note here if you do not want to get notifications
in the future if your translation needs an update.
If there isn't anything else to say, please delete the heading.
```

Author: That's a good idea! Do you keep track of who wants to be notified elsewhere then?

Reviewer: Let's see if that's really necessary. I guess most people will be OK with being contacted. If a translation needs an update, we can create a new issue accordingly and just look at the past PRs as we will need the GitHub handle of the user anyways.
